### PR TITLE
Enhance search feature to support `path`

### PR DESCRIPTION
**Pull request by bpateldx**
- This allows easy and quick navigation for well-known end-points by `path` or keywords in a path 
- Supports existing search functionality

## Testing
Added E2E Test for the change
## Screenshots
image
image
**_Now search work for keywords in path:_**
image
**_Now search works for paths:_**
image
**_Current search still works:_**
image

## Completed

- [ x] Code is linted
- [ x] Tested
- [ x] All new/updated code is covered with (E2E) tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,615 @@
+# 2.0.0-rc.76 (2022-08-30)
+
+
+### Bug Fixes
```
Comment (AlexVarchuk) [inline CHANGELOG.md:4]: please remove this file from PR.

**Review by AlexVarchuk — CHANGES_REQUESTED** [review]

**Comment by AlexVarchuk** [comment]
Hi @bpateldx, thank you for your contribution. Please, add some screenshots of how the search is working now. Thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (bpateldx) [reply CHANGELOG.md]: Updated PR with additional screenshots

Comment (AlexVarchuk) [reply CHANGELOG.md]: @bpateldx My bad. I meant to remove changes in this file from PR.
sorry for the inaccuracy

Comment (bpateldx) [reply CHANGELOG.md]: @AlexVarchuk Just to be clear, you're simply asking to revert change log file back to your master version, correct? I assume you would be updating the changelog file upon release.

Comment (AlexVarchuk) [reply CHANGELOG.md]: @bpateldx yes, please revert `CHANGELOG.md` file to the master version. We'll update it when making the next release. Thx

Comment (bpateldx) [reply CHANGELOG.md]: Done, please review.

**Review by AlexVarchuk — APPROVED** [review]

**Comment by bpateldx** [comment]
@AlexVarchuk Thank you for your feedback on reviews. What's ETA/process on merging the PR.

**Pull request merged**